### PR TITLE
Feature/side navigation

### PR DESCRIPTION
**Pull request by itsnameissteven**
## Is this a fix or a feature?
This is a feature
## What is the change?
* Hamburger Icon has been added to the app
  * This can live on the left or right side of screen
* Menu toggles open and closed between clicks
* Users can travel to different links from menu
* Tests added for user flow.
* Temp image added to the landing screen
## What does it fix?
n/a
## Where should the reviewer start?
* Review the new Menu component and tests
## How should this be tested?
toggle the menu open and close, click links to make sure all functionality is there OR run cypress to automatically check functionality
## Relevant ScreenShots
Screen Shot 2021-05-17 at 11 02 52 AM

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -23,30 +24,36 @@ const App = () => {
...
 
-  const formProps = {name, setName, username, setUsername, email, 
+  const formProps = {name, setName, email, 
     setEmail, page, setPage, make, setMake, model, setModel, year, setYear, origin, 
     setOrigin, destination, setDestination, days, setDays, departTime, setDepartTime, 
     password, setPassword, bio, setBio
   };
 
+  const toggleMenu = () => {
+    console.log('es')
```
Comment (PaigeVannelli) [inline src/components/App/App.tsx:38]: Let's remember to take out our console logs!

**Review by PaigeVannelli — APPROVED** [review]

File: src/components/Menu/Menu.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+interface MenuProps {
+  toggleMenu: () => void;
+  menuOpen: boolean
+}
+
+const Menu: FC<MenuProps> = ({ menuOpen, toggleMenu }) => {
+
+  return (
+    <div className={menuOpen ? 'menu' : 'menu-closed'} onClick={toggleMenu}>
```
Comment (alexthompson207) [inline src/components/Menu/Menu.tsx:12]: This component looks great! I really like the different routes to each page!

File: cypress/integration/hitch-login_spec.js [hunk 3]
```diff
@@ -592,14 +556,50 @@ describe('User time and day input, and about me section', () => {
...
+    cy.get('button').eq(1).click();
+
+    cy.get(".registration__back-btn").contains('Back');
+
+    cy.get(".registration__back-btn").click();
+
+    cy.get('input').should('have.length', 3);
+    cy.get('.register-form__header').contains('Name');
+  })
+})
```
Comment (alexthompson207) [inline cypress/integration/hitch-login_spec.js:605]: Thanks for updating the tests, I think we pretty much have all happy/ sad paths covered!

**Review by alexthompson207 — APPROVED** [review]
Really great work with this, the menu component looks great! You're a styling genius!!!

File: src/components/App/App.tsx (hunk 1 above)
Comment (alexthompson207) [reply src/components/App/App.tsx:38]: Good catch

**Pull request merged**